spec/jobs/reports/monthly_key_metrics_report_spec.rb

```diff
@@ -38,9 +39,9 @@
       ['Metric', 'Trailing 30d', 'Trailing 60d', 'Trailing 90d'],
     ]
   end
-  let(:mock_proofing_report_data) do
+  let(:mock_monthly_idv_data) do
     [
-      ['metric', 'num_users', 'percent'],
+      ['Metric', 'June 2024', 'July 2024', 'August 2024'],
     ]
```

Comment:
This is a confusing diff right here because the red and green are unrelated. Here, I:

- Removed `mock_proofing_report_data` because absolutely nothing used it
- Added `mock_monthly_idv_data` as mocked data (used on 63-64 below)